tests/rules/test_clock.py

```diff
@@ -46,7 +44,7 @@ async def aput(self, dt):
         self.data.private_origin = self.salinfo.domain.origin
         setattr(self.data, f"{self.salinfo.name}ID", self.salinfo.index)
         self._writer.write(self.data)
-        await asyncio.sleep(0.001)
+        await asyncio.sleep(0)
```

Comment:
I see in many other unit tests that you removed `await asyncio.sleep(0.001)` but here you set the sleep time to zero. What effect does that have?

Reply:
Good catch! In fact it does nothing useful here. Removed.